src/app/teacher/manage-tags-dialog/manage-tags-dialog.component.ts

```diff
@@ -65,17 +65,8 @@ export class ManageTagsDialogComponent implements OnInit {
     this.subscriptions.unsubscribe();
   }
 
-  protected create(): void {
-    this.dialog.open(CreateTagDialogComponent, {
-      panelClass: 'dialog-md'
-    });
-  }
-
-  protected edit(tag: Tag): void {
-    this.dialog.open(EditTagDialogComponent, {
-      panelClass: 'dialog-md',
-      data: tag
-    });
+  protected cancel(tag: Tag): void {
```

Comment:
The function name cancel is a bit vague here. Is there a more informative name, like cancelEditing?